Fallback to `/tmp` if env `XDG_RUNTIME_DIR` is not set.

--- a/src/util/anon_file.c
+++ b/src/util/anon_file.c
@@ -136,6 +136,11 @@
    char *name;
 
    path = getenv("XDG_RUNTIME_DIR");
+#ifdef __linux__
+   if (!path) {
+      path = "/tmp";
+   }
+#endif
    if (!path) {
       errno = ENOENT;
       return -1;
